### PR TITLE
Solves console log issues

### PR DESCRIPTION
**Pull request by TG1999**
This patch includes:
- [ ] A relevant test
- [ ] A relevant documentation update

### CODE REVIEW

**Comment by TG1999** [comment]
Please have a look @saschanaz

File: webpack.config.js [hunk 1]
```diff
@@ -10,5 +10,17 @@ module.exports = {
     globalObject: "this"
   },
   mode: "production",
-  devtool: "source-map"
+  devtool: "source-map",
+    optimization: {
```
Comment (saschanaz) [inline webpack.config.js]: First we have to make indentations here consistent (it's over-indented now), and then you'll soon find what's wrong 🤣

Comment (TG1999) [reply webpack.config.js]: 😂😂 okay I will revise it, let me see what's wrong in it

File: webpack.config.js [hunk 2]
```diff
@@ -10,5 +10,17 @@ module.exports = {
...
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          output: {
+            keep_classnames: true
+          }
+        }
+      )
```
Comment (saschanaz) [inline webpack.config.js]: Oops, seems I confused you.. this was the problem:

```suggestion
      })
```

File: webpack.config.js (hunk 1 above)
Comment (saschanaz) [reply webpack.config.js]:
```suggestion
  optimization: {
```

This is indeed over-indented, though 😉

**Comment by TG1999** [comment]
@saschanaz its still failing

File: webpack.config.js [hunk 3]
```diff
@@ -10,5 +10,17 @@ module.exports = {
     globalObject: "this"
   },
   mode: "production",
-  devtool: "source-map"
+  devtool: "source-map",
+    optimization: {
+    minimizer: [
+      new TerserPlugin({
+        terserOptions: {
+          output: {
```
Comment (saschanaz) [inline webpack.config.js]: Could you try removing this `output: {}` and replace it with the inner `keep_classnames: true`?

Comment (TG1999) [reply webpack.config.js]: Yeah sure

Comment (TG1999) [reply webpack.config.js]: I have made the following changes

**Comment by TG1999** [comment]
Now lets start working on the test part :fire:

**Comment by TG1999** [comment]
Please guide me how should I write the test part ?

**Comment by saschanaz** [comment]
From https://github.com/w3c/webidl2.js/issues/427#issuecomment-536296433

>The error object we exported in #428 should have err.constructor.name === "WebIDLParseError" but currently #415 makes it _, so we want to fix it.

So I think we can add a line on the test we wrote in #428.

**Comment by TG1999** [comment]
> From [#427 (comment)](https://github.com/w3c/webidl2.js/issues/427#issuecomment-536296433)
> 
> > The error object we exported in #428 should have err.constructor.name === "WebIDLParseError" but currently #415 makes it _, so we want to fix it.
> 
> So I think we can add a line on the test we wrote in #428.

Yeah should I write in catch statement then?

**Comment by saschanaz** [comment]
Yup

**Comment by TG1999** [comment]
@saschanaz tests are also passing looks good to be merged what say?

File: test/invalid.js [hunk 4]
```diff
@@ -136,6 +136,7 @@ describe("Error object structure", () => {
       throw new Error("Shouldn't reach here");
     } catch (err) {
       expect(err).toBeInstanceOf(WebIDLParseError);
+      expect(err.constructor.name).toEqual("WebIDLParseError");
```
Comment (saschanaz) [inline test/invalid.js]:
```suggestion
      expect(err.constructor.name).toBe("WebIDLParseError");
```

**Comment by TG1999** [comment]
Okay now any more issues on which I can work with you or any new feature I can implement ? :smile:

**Comment by saschanaz** [comment]
Hmm, remaining issues requires some Web IDL specific knowledge, are you interested?

It's time to bed here, you may read the [specification document](https://heycam.github.io/webidl/) in the meantime and try understanding what the existing code does in terms of the spec 😀

**Comment by TG1999** [comment]
I will like to learn the working of web standards and contribute in it so will like to do anything related to it. I will read the docs in the meantime and come back to you when done.

**Pull request merged**